tests/Jobs/GenerateImageAnnotationPatchTest.php

```diff
@@ -29,6 +29,7 @@ public function testHandleStorage()
         Storage::fake('test');
         $image = $this->getImageMock();
         $annotation = ImageAnnotationTest::create();
+        $annotation->shape_id = Shape::pointId();
```

Comment:
This is done like this (otherwise the new ID is not saved in the database):

```suggestion
        $annotation = ImageAnnotationTest::create([
            'shape_id' => Shape::pointId(),
        ]);
```